runtime/gc_vlhgc/WriteOnceCompactor.cpp

```diff
@@ -1182,6 +1167,36 @@ MM_WriteOnceCompactor::fixupObjectsInRange(MM_EnvironmentVLHGC *env, void *lowAd
 	}
 }
 
+MMINLINE void
+MM_WriteOnceCompactor::preObjectMove(MM_EnvironmentVLHGC* env, J9Object *objectPtr, UDATA *objectSizeAfterMove)
+{
+	*objectSizeAfterMove = _extensions->objectModel.getConsumedSizeInBytesWithHeaderForMove(objectPtr);
+	_extensions->objectModel.preMove(env->getOmrVMThread(), objectPtr); // To be deprecated
+	//env->preObjectMoveForCompact(objectPtr); // Will substitute line above
```

Comment:
seems like there is an action due here?

Reply:
I thought to leave these here since they'll be part of the 3rd part of changes. Or should I remove it?

Reply:
it's ok if this is to be addressed in followup changes

Reply:
Yes they are!